src/transaction/combine.ts

```diff
@@ -1,42 +1,80 @@
 import * as _ from 'lodash'
 import binary from 'ripple-binary-codec'
-import * as utils from './utils'
 import BigNumber from 'bignumber.js'
+import {ValidationError} from '../common/errors'
 import {decodeAccountID} from 'ripple-address-codec'
 import {validate} from '../common'
 import {computeBinaryTransactionHash} from '../common/hashes'
+import {JsonObject} from 'ripple-binary-codec/dist/types/serialized-type'
+
+//The transactions should all be equal EXCEPT for the Signers field
+function validateAllSignedTransactionsAreEqual( transactions : Array<JsonObject> ) {
+  const tx = transactions[0];
+  const _txSignersTemp = tx.Signers;
```

Comment:
leading underscore is confusing and indicates that the variable is not going to be used, but it will